java/scripts/sign_and_bundle_release.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ];
+then
+      echo -e "Usage: $0 \e[4mVERSION\e[0m"
+      echo -e "where \e[4mVERSION\e[0m is part of the tag release/java/<version>, ex v1.0.0"
+      exit 1
+fi
+
+
+echo "downloading latest release from github"
+RELEASE_INFO=$(curl -s -H "Accept: application/vnd.github+json" https://api.github.com/repos/loosebazooka/protobuf-specs/releases/tags/release/java/$1)
+# https://github.com/loosebazooka/protobuf-specs/releases/tag/release/java/v1.2.3
+RELEASE_VERSION="$(echo $RELEASE_INFO | jq -r '.tag_name')"
+RELEASE_DIR="${RELEASE_VERSION//\//_}"
+
+echo "release version is: ${RELEASE_VERSION}"
+
+if [ -d $RELEASE_DIR ]; then
```

Comment:
`"$RELEASE_DIR"`?

Maybe run this whole thing through a shell linter, there's 10 million ways for shell scripts to just die for no reason because you forgot to make an offering to the bash gods